docs/documentation/indexing/token_filters.mdx

```diff
@@ -21,6 +21,9 @@ paradedb.tokenizer('default', lowercase => false, remove_long => 255)
   Applies language-specific stemming to each token. See
   [stemming](/documentation/indexing/token_filters) for supported languages.
 </ParamField>
+<PipelineField body="stop_words" default={NULL}>
```

Comment:
Tantivy has two stop word filters: prebuilt language ones and user-defined ones. We should expose both as separate options.

Reply:
Tantivy defines the stemmer and stop word filters separately and we should too. Stemming removes letters at the end of words, whereas stop words remove words altogether. Plus if we did that, then the user would only be able to use one of the language stemmers and the stop word stemmers.